### ISSUE TITLE
F_ jacobian and L_ jacobian is wrong

### ISSUE THREAD

**Issue by happain** [bolderflight/uNavAHRS#5]
hi, i think you F_ and L_ maybe wrong, here is my F_ and L_, F_(1,5), F_(1,6), F_(3,6), L_(4,0), L_(5,1), L_(6,2)
      F_(0,4) = 0.5*_dt*x_(1,0);       F_(0,5) = 0.5*_dt*x_(2,0);       F_(0,6) = 0.5*_dt*x_(3,0);
      F_(1,4) = -0.5*_dt*x_(0,0);      F_(1,5) = 0.5*_dt*x_(3,0);      F_(1,6) = -0.5*_dt*x_(2,0);
      F_(2,4) = -0.5*_dt*x_(3,0);      F_(2,5) = -0.5*_dt*x_(0,0);      F_(2,6) = 0.5*_dt*x_(1,0);
      F_(3,4) = 0.5*_dt*x_(2,0);       F_(3,5) = -0.5*_dt*x_(1,0);      F_(3,6) = -0.5*_dt*x_(0,0);
      F_(4,4) = 1.0;                   F_(5,5) = 1.0;                   F_(6,6) = 1.0;
      // process noise matrix
      L_(4,0) = -1.0;               L_(4,1) = 0.0;               L_(4,2) = 0.0;
      L_(5,0) = 0.0;               L_(5,1) = -1.0;               L_(5,2) = 0.0;
      L_(6,0) = 0.0;               L_(6,1) = 0.0;               L_(6,2) = -1.0;

**Comment by Nxt-pl** [comment]
Have you got any references how to construct these jacobians?

**Comment by happain** [comment]
> Have you got any references how to construct these jacobians?

yes, i think F_ and L_ is wrong, i find some paper and open source code, these jacobians is the same as mine

**Comment by Tarek-H** [comment]
Hello did you resolve the problem ? , i didnt change anything , it work but the Yaw and heading  drift  quickly when the IMU  move    , is like someting is wrong .
